### ISSUE TITLE
Sliders produce by direct point manipulation overlap excessively in 2.13 update

### ISSUE THREAD

**Issue by JacobSmall** [DynamoDS/Dynamo#12600]
## Dynamo version
2.13

## Operating system
Windows 10 - 21h2

## What did you do? 
Moved a point with the geometry manipulation method, to introduce number sliders

## What did you expect to see?
Number sliders added to the canvas, with spacing and locations functionally placed for renaming and manipulation; similar to what we get with autocomplete but with easier value selection.

## What did you see instead?
Nodes mostly on top of each other, bounding boxes limited to the title. See attached GIF.
![Overlaping nubmer sliders](https://user-images.githubusercontent.com/29869395/152086575-5f977dff-cad3-4115-b4c3-5494c88d89c1.gif)

**Comment by Amoursol** [comment]
@reddyashish looks like we could apply the changes made to the Node Autocomplete portion back to this :) ? 

FYI @QilongTang

**Comment by reddyashish** [comment]
Yeah, created a Jira task to look into it: https://jira.autodesk.com/browse/DYN-4561
